Makefile

```diff
@@ -98,10 +98,10 @@ test-cpp:
 	cmake -BBuildTests -G Ninja -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTS=ON -DENABLE_WARNINGS=ON -DPL_BACKEND=$(PL_BACKEND) $(OPTIONS)
 ifdef target
 	cmake --build ./BuildTests $(VERBOSE) --target $(target)
-	OMP_PROC_BIND=false ./BuildTests/$(target)
+	./BuildTests/$(target)
 else
 	cmake --build ./BuildTests $(VERBOSE)
-	OMP_PROC_BIND=false cmake --build ./BuildTests $(VERBOSE) --target test
+	cmake --build ./BuildTests $(VERBOSE) --target test
```

Comment:
Why is this removed? Don't we want to silence the Kokkos warning from this? Is the env var set elsewhere?

Reply:
I found this to slow down the C++ tests considerably on many-core CPUs, so I thought I would just let the developer define the variable as he sees fit. The Kokkos warning does not annoy me that much, but I don't mind leaving it and unsetting myself when suitable.

Reply:
I'm fine leaving it out then. :+1: